ui/src/FindBar.tsx

```diff
@@ -92,12 +133,59 @@ export class FindBar extends React.PureComponent<{}, FindBarState> {
     this.dispatchEventToPdfJs("find");
   }
 
-  handleNextButtonClick() {
-    this.dispatchEventToPdfJs("findagain");
+  wrapIndex(index: number, len: number) {
+    return (index % len + len) % len;
   }
 
-  handlePreviousButtonClick() {
-    this.dispatchEventToPdfJs("findagain", true);
+  selectSymbol(id: string, boxId: string) {
+    this.context.setSelectedEntity(id, "symbol");
+    this.context.setSelectedAnnotationId(convertToAnnotationId(id));
+    this.context.setSelectedAnnotationSpanId(boxId);
+  }
+
+  unselectSymbol() {
```

Comment:
nit pick for consistency with terminology elsewhere in the code base

```suggestion
  deselectSymbol() {
```